neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -190,6 +191,47 @@ def get_project(self, project_id: str) -> Project:
         except HTTPNotFound:
             raise ProjectNotFound(project_id)
 
+    @with_api_exceptions_handler
+    def get_available_projects(self,
+                               workspace_id: Optional[str] = None,
+                               search_term: Optional[str] = None
+                               ) -> List[Project]:
+        try:
+            response = self.backend_client.api.listProjects(
+                limit=5,
+                organizationIdentifier=workspace_id,
+                searchTerm=search_term,
+                sortBy=['timeOfCreation'],
```

Comment:
Shouldn't we rely on `last_experiment_creation_time` instead?

Reply:
I've talked with Marcin and the expected behavior is to sort by both last experiment creation and project creation :wink:

Reply:
I've changed it to `lastViewed` due to lack of other options in `sortBy` argument without API updates. Approved by Marcin.